tests/python/unittest/test_runtime_graph_debug.py

```diff
@@ -185,5 +191,47 @@ def check_remote(server):
     check_remote(rpc.Server("127.0.0.1"))
 
 
+@tvm.testing.requires_llvm
+def test_run_single_node(graph):
+    mlib_proxy = tvm.support.FrontendTestModule()
+    mlib_proxy["myadd"] = myadd
+    try:
+        mod: debug_executor.GraphModuleDebug = debug_executor.create(graph, mlib_proxy, tvm.cpu(0))
+    except ValueError:
+        return
```

Comment:
Seems like we should throw an exception if there is an error.

Reply:
This will be hit if TVM is not built with the profiler on. Added a comment clarifying

Reply:
Its better to use `@pytest.skipif(tvm.support.libinfo()["USE_PROFILER"] != "ON", "TVM was not built with profiler support")`

Reply:
Cool, done